### ISSUE TITLE
Current master bits: es5 bundle breaks due to non-transpiled dependencies

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#2559]
## Describe the bug
Web Chat does not work in IE11; with errors:
![image](https://user-images.githubusercontent.com/14900841/68166075-0c2ab680-ff16-11e9-80f1-4175b042023b.png)

## To Reproduce

1. Run 01.b.getting-started-es5 sample
2. Open in Internet Explorer
3. Open console
4. See errors match screenshot above

## Expected behavior

Web Chat should work in IE11

## Additional context

It looks like the bump of `remark` caused this issue due to[ `is-plain-obj` not being transpiled.](https://github.com/sindresorhus/is-plain-obj/issues/6) `unified`, a child of `remark`, uses `is-plain-obj` even though the package is intended for node environments, not the browser. To get rid of this problem we'll have to find which version of remark doesn't ultimately use `is-plain-obj` version 2.0

TL:DR; de-bump `remark`

[Bug]

**Comment by corinagum** [comment]
Fixed by #2578
